pkg/domain/entities/container_ps.go

```diff
@@ -0,0 +1,167 @@
+package entities
+
+import (
+	"sort"
+	"strings"
+
+	"github.com/containers/libpod/cmd/podman/shared"
+	"github.com/containers/libpod/libpod"
+	"github.com/cri-o/ocicni/pkg/ocicni"
+	"github.com/pkg/errors"
+)
+
+// Listcontainer describes a container suitable for listing
+type ListContainer struct {
+	// Container command
+	Command []string
+	// Container creation time
+	Created int64
+	// If container has exited/stopped
+	Exited bool
+	// Time container exited
+	ExitedAt int64
```

Comment:
Why int64? Shouldn't this be a time.Time?

Reply:
that is what it was before i moved it. i am not changing any of the types in the struct.